index.html

```diff
@@ -4550,13 +4550,13 @@ <h2>Immutability</h2>
 <code>type</code> after it is defined? Or for a key to change its value? Or
 would it be better to require a new <code><a>id</a></code> when certain
 fundamental properties of an object change? Malicious takeovers of a website
-often aim for an outcome where the site keeps its identifier (the host name),
+often aim for an outcome where the site keeps its host name identifier,
 but is subtly changed underneath. If certain properties of the site were
-required by the specification to be immutable (for example, the <a
+required by the specification to be immutable, such as the <a
 target="_blank"
 href="https://en.wikipedia.org/wiki/Autonomous_system_(Internet)">ASN</a>
-associated with its IP address), such attacks might be much harder and more
-expensive to carry out, and anomaly detection would be easier.
+associated with its IP address, anomaly detection would be easier and there
+might be attacks that would be much harder and more expensive to carry out.
```

Comment:
```suggestion
but is subtly changed underneath. If certain properties of the site, such
as the <a target="_blank"
href="https://en.wikipedia.org/wiki/Autonomous_system_(Internet)">ASN</a>
associated with its IP address, were required by the specification to be
immutable, anomaly detection would be easier, and attacks would be much
harder and more expensive to carry out.
```

Reply:
Fixed in 2d237be3061dcb8c6f0df1f02459917e505dee1a.

@TallTed - I couldn't merge this because you can't merge change suggestions made to deleted lines... or, so Github tells me.